CHANGELOG.md

```diff
@@ -1,3 +1,74 @@
+# 3.10.1 Belgian Tripel (2019-04-17)
+
+### Content Views
+ * Content View Version - ostree branch - generates error ([#26514](https://projects.theforeman.org/issues/26514), [9b4e10eb](https://github.com/Katello/katello.git/commit/9b4e10eb4e3fe88d662de54781cc683a17d40f60))
```

Comment:
On second look with the 3.11.1, should we have a heading Bug Fixes here? Similar to https://github.com/Katello/katello/pull/8085/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3

Reply:
Nice - I removed it accidentally. Fixed!

Reply:
ACK again :smile: